packages/interpreter/src/helpers/index.ts

```diff
@@ -132,6 +132,15 @@ export function getOperationValue(
     case "||":
       return operands.left || operands.right;
 
+    case '**':
+      if (checkNumberOperands(operands)) {
+        return Math.pow(operands.left,operands.right);
+      }
+      throw exception;
```

Comment:
How about throwing with meaningful message, unless that is part of exception already.

Reply:
it already throws a meaningful message. try it out in the playground.

Reply:
![image](https://user-images.githubusercontent.com/48474744/158075099-0f50c73b-ef2f-4344-be1d-6219b4b81f21.png)

Meaningful message is already there.